### ISSUE TITLE
Line is too long?

### ISSUE THREAD

**Issue by AhmedCode99** [ubsuny/23-Homework4G3#52]
pylint is returning a line is too long error. How can a line be too long?

The code runs fine btw

**Comment by reshnashrestha** [comment]
The suggestion might be focusing on the specific code guidelines for number of characters on the line.

**Comment by tirthbha** [comment]
@AhmedCode99 Yeah, you can set the minimum number of characters using a single command to the code.

**Comment by tirthbha** [comment]
@AhmedCode99 By default, it is around 79, I guess.

**Comment by yasmensarhan27** [comment]
the maximum length is 70 char 
that's why you get this error

**Comment by tirthbha** [comment]
@yasmensarhan27 Yeah, but can set that length to a particular value, right? So that it won't show any error while running the code regarding the line length.

**Comment by s4il3sh** [comment]
> pylint is returning a line is too long error. How can a line be too long?
> 
> The code runs fine btw

You can shift your long code to next line using blackslash " \ ". It will work like a charm!

**Comment by Mosaddeq107** [comment]
@s4il3sh  Does adding a '\' in a code works? Or is it applicable to characters only?
